workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -12,6 +12,9 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Receives incoming ops from the SpatialOS runtime and dispatches them.
```

Comment:
Messages rather than ops I think. I don't think you need to say it dispatches them. Especially as it's not wholly accurate.